data/sql/updates/pending_db_world/rev_1679709941852342200.sql

```diff
@@ -0,0 +1,5 @@
+--
+-- Repair Pathaleon the Calculator Loot Table
+UPDATE `reference_loot_template` SET `GroupId`='5' WHERE `Entry`=35005 AND `Item` IN (28285, 28278, 28286, 28288, 28275, 27899);
```

Comment:
just some nit-picks, dw.
But no single-quotes on integers

```suggestion
-- Repair Pathaleon the Calculator Loot Table
UPDATE `reference_loot_template` SET `GroupId`=5 WHERE `Entry`=35005 AND `Item` IN (28285, 28278, 28286, 28288, 28275, 27899);
```